.env.sample

```diff
@@ -0,0 +1,2 @@
+PROVIDER_URL=http://localhost:8545
+IPFS_GATEWAY_URL=https://ipfs.io
```

Comment:
In the frontend repo this file is called `.env.example`. Do we use `sample` in other places? No big deal but would be nice to sync them up since it is just a tiny effort